### DISCUSSION TITLE
Regarding GSOC 2023 project contributions

### DISCUSSION THREAD

**Discussion by sai-praveenkumar** [cncf/mentoring#762] [Q&A] [answered]
Hello everyone! I am SaiPraveenKumar.
I am a final year CS student.
I am new to open source and I'm really interested and excited about the upcoming projects offered by CNCF and I want to contribute to these projects in the upcoming GSOC 2023 as well.
The technologies I know are : JAVA,PYTHON, JAVASCRIPT, HTML, CSS.
It would be really helpful if you could guide me towards contributing to the CNCF community so that I could get started to contribute to the existing as well as the upcoming projects offered by CNCF asap.
Looking forward to collaborate with you.
Thank You

**Comment by johurul000** [comment, marked as answer]
I don't know much as i am new too. But if you want to contribute to core parts of the projects in cncf, learning go lang Will be very helpful as lots of those cncf projects us go lang.

**Comment by aliok** [comment]
hi @sai-praveenkumar 

You can have a look at the answer here for getting started with contributions: https://github.com/cncf/mentoring/discussions/755
